pybaseball/analysis/trajectories/batted_balls/parameters.py

```diff
@@ -0,0 +1,73 @@
+import attr
+import numpy as np
+from pybaseball.datahelpers.postprocessing import check_greater_zero
+
+
+@attr.s(frozen=True, kw_only=True)
+class BattedBallConstants:
+    mass = attr.ib(
+        default=5.125, validator=check_greater_zero, metadata={"units": "oz"}
+    )
+    circumference = attr.ib(
+        default=9.125, validator=check_greater_zero, metadata={"units": "in"}
+    )
+
+
+@attr.s(frozen=True, kw_only=True)
+class DragForceCoefficients:
+    cd0 = attr.ib(default=0.3008)
+    cdspin = attr.ib(default=0.0292)
+
+
+@attr.s(frozen=True, kw_only=True)
+class LiftForceCoefficients:
+    cl0 = attr.ib(default=0.583)
+    cl1 = attr.ib(default=2.333)
+    cl2 = attr.ib(default=1.120)
+    tau = attr.ib(default=10000, metadata={"units": "seconds"})
+
+
+@attr.s(kw_only=True)
+class EnvironmentalParameters:
+    # environmental parameters
+    g_gravity = attr.ib(default=32.174, metadata={"units": "ft_per_s_per_s"})
+    vwind = attr.ib(default=0, metadata={"units": "mph"})  # mph
+    phiwind = attr.ib(default=0, metadata={"units": "deg"})  # deg
+    hwind = attr.ib(default=0, metadata={"units": "ft"})  # ft
+    relative_humidity = attr.ib(default=50)
+    pressure_in_hg = attr.ib(default=29.92)
+    temperature_f = attr.ib(default=70, metadata={"units": "F"})  # F
+    elevation_ft = attr.ib(default=15, metadata={"units": "ft"})
+    beta = attr.ib(
+        default=1.217e-4, validator=check_greater_zero, metadata={"units": "per_meter"}
+    )
```

Comment:
Do we use the `{"units": ... }` anywhere?

Reply:
no, they effectively play the role of comments / documentation